package.json

```diff
@@ -16,40 +16,46 @@
   },
   "repository": "https://github.com/percy/ember-percy",
   "engines": {
-    "node": ">= 0.12.0"
+    "node": "^4.5 || 6.* || >= 7.*"
```

Comment:
Related to comment in travis.yml, we'd like to maintain backwards compatibility for Node 4.x releases if possible. Can you explain a bit more about this line change?

Reply:
I pulled this engine setup from here: https://github.com/ember-cli/ember-cli/blob/master/package.json#L148

I've generally seen this mentioned as the recommendation by ember-cli and for ember projects/add-ons. I believe the idea is to support LTS versions of Node, but with the preference of a more stable version of 4.